apps/site/src/content/docs/cypherpunk.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Cypherpunk Hackathon
+description: Samui Wallet participates in the Solana Cypherpunk Hackathon by Colosseum.
+---
+
+Samui Wallet participates in the Solana Cypherpunk Hackathon by Colosseum.
```

Comment:
Should we mention when we started this project or the date of the hackathon? I think it might be useful so visitors know we are early into this.